console/algorithms/Cargo.toml

```diff
@@ -12,10 +12,15 @@ path = "benches/bhp.rs"
 harness = false
 
 [[bench]]
-name = "poseidon_sponge"
+name = "poseidon"
```

Comment:
This will require a change to the benchmarks CI, need to update the benchmark being run from `poseidon_sponge` to `poseidon`.